aiomisc/utils.py

```diff
@@ -22,11 +22,12 @@ def chunk_list(iterable: Iterable[Any], size: int):
         item = list(itertools.islice(iterable, size))
 
 
-OptionsType = List[Tuple[int, int, int]]
+OptionsType = Tuple[Tuple[int, int, int], ...]
 
 
-def bind_socket(*args, address: str, port: int, options=(),
-                reuse_addr=True, reuse_port=False):
+def bind_socket(*args, address: str, port: int, options: OptionsType = (),
+                reuse_addr: bool = True, reuse_port: bool = False,
+                proto_name: str = 'tcp'):
```

Comment:
Maybe it's better to define enum for `proto_name`?

Reply:
No. The proto name is user defined.